lib/Distribution/Sle/15_current.pm

```diff
@@ -28,6 +28,8 @@ use Installation::RepositoryURL::RepositoryURLController;
 use Installation::AddOnProductInstallation::AddOnProductInstallationController;
 use Installation::SystemRole::Sle::SystemRoleController;
 use Installation::ModuleRegistration::ModuleRegCodeController;
+use YaST::DNSServer::DNSServerSLEController;
+use YaST::DNSServer::DNSServerSetupSLEController;
```

Comment:
should be `YaST::DNSServer::Sle::Controller` we tried in the past to be consistent with CPAN module naming. The `Sle` in the middle indicate that we need another folder to store. Please follow that, there are other examples that I pointed offiline.